FAQ.md

```diff
@@ -0,0 +1,91 @@
+- `allow-popups-to-escape-sandbox`: This removes the sandbox restriction from windows opened from the sandboxed document.
+
+> Note that we do not set `allow-same-origin` flag and hence the iframe will have an origin of `null`.
+
+## May I open modal overlays on top of the entire Contentful page?
+
+No, the iframe sandboxing restrictions will prevent you from doing so.
+
+## May I open regular javascript popups from an extension?
+
+Yes, you can use `window.open()` and links with `target="_blank"`
+
+## Is it mandatory to host my extensions on HTTPS?
+
+We do not allow you to reference non-HTTPS URLs (like `http://`) with the exception of URLs starting with `http://localhost` for development purposes.
+If you serve the extension from `localhost` you need to disable [mixed content protection in the browser](https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox, https://support.google.com/chrome/answer/1342714).
+
+## Is CORS required?
+
+As the extension is run in a sandboxed Iframe it receives the origin “null”.
+Therefore, if you’re using custom web fonts, things don't work as expected as they don’t play well with sandboxed iframes at the moment.
+
+To overcome this, you can configure your servers to send the following headers:
+`Access-Control-Allow-Origin` "null" or  `Access-Control-Allow-Origin` "*"
+
+## What happens when deploying a broken extension?
+
+If the URL defined in your `extension.json` is invalid or contains errors, then the extension will either not show up or show as broken.
+
+## How do you debug extensions?
+
+Most of [our examples](https://github.com/contentful/bundles/tree/master/extensions) demonstrate how to deploy an extension running on a local web server, so you can use your browser console to debug it the same way you debug any other web application.
+
+## With which permissions does an extension run?
+
+An extension runs with the permissions of the user displaying it, not with the ones from the user publishing the extension itself. This means the extensions needs to provide a fallback behaviour in case it tries to perform an action the current user is not allowed to do.
+
+## What are the guidelines for choosing extension names?
+
+There is no strong validation of extension names, but remember that long ones will break the layout.
+
+## Can user provide a extension thumbnail?
+
+This is not possible right now, but we’re working on adding this possibility in the future.
+
+## What happens with the content once a extension is removed?
+
+The content is independant from the extension that is used to edit it. When you remove an extension the content is still there and you can choose another appearance from the field’s settings.
+
+## Can the extension destroy my content?
+
+In theory yes, since they have access to all the content the user that is running the extension has. Make sure you implement safeguards to avoid losing your content.
```

Comment:
Imho, "losing your content" sounds more impactful. And that might be the right tone for this message.